docs/contributing/about-platform-toolkit-and-library-names.md

```diff
@@ -22,61 +21,99 @@ for wxWidgets, but is encompassing enough to maintain a relatively complete
 set of cross platform build tools on a single machine and to provide an
 obvious slot for new ports to slip into.
 
+The libraries use the following naming convention.
+When an item of the name is empty, `'_'` or `'-'` are omitted.
 
-For UNIX libraries, the canonical library name shall be of the form:
+*UNIX libraries* (using configure):
 
-	libwx_$(toolkit)$(widgetset)$(debug)-$(version)-$(host).$(lib_extension)
+    libwx_$(toolkit)$(widgetset)$(unicode)_$(flavour)_$(name)-$(version)-$(host).$(lib_extension)
 
-For MSW (native hosted only) libraries the library name should be of
-the form:
+*Windows libraries* (using VS solution or makefile.gcc/vc):
```

Comment:
Perhaps mention that it applies also for CMake when the generator is any of the Visual Studio or NMake and MinGW Makefiles?

Reply:
Good point. I forgot to mention CMake in this document.